src/base/utils/looks_like.jl

```diff
@@ -0,0 +1,34 @@
+"""
+    looks_like_exchange_reaction(rxn_id::String;
+        exclude_biomass = false,
+        biomass_strings = _constants.biomass_strings,
+        ex_prefixes = _constants.exchange_prefixes,
+    )
+
+Returns a predicate function that can be used to identify reactions that are probably 
+exchange or biomass reactions. Exchange reactions are identified based on matching prefixes 
+in the set `ex_prefixes` and biomass reactions are identified by looking for occurences 
+of `biomass_strings` in the reaction id.
+
+# Example
+```
+filter(is_probably_exchange_reaction, reactions(model)) # returns strings
+findall(is_probably_exchange_reaction, reactions(model)) # returns indices
+
+# to use the optional arguments you need to instantiate the function
+filter(x -> is_probably_exchange_reaction(x; exclude_biomass=true), reactions(model)) # returns strings
+findall(x -> is_probably_exchange_reaction(x; exclude_biomass=true), reactions(model)) # returns indices
+```
+"""
+function looks_like_exchange_reaction(rxn_id::String;
+    exclude_biomass = false,
+    biomass_strings = _constants.biomass_strings,
+    ex_prefixes = _constants.exchange_prefixes,
+)   
+    if any(startswith(rxn_id, x) for x in ex_prefixes) && !any(occursin(x, rxn_id) for x in biomass_strings) # found exchange reaction
+        return true
+    elseif !exclude_biomass && any([occursin(x, rxn_id) for x in biomass_strings]) # biomass
+        return true
+    end
+    return false
+end
```

Comment:
I still don't get why we would want to use this to find biomass reactions? Now, if a user would input a model that has a biomass exchange ("EX_biomass") and calls with `exclude_biomass = false`, they would get both the biomass exchange and the biomass reaction, right?

Reply:
Yeah, that is how it currently works. Ah, I am confused about the behaviour you want? Do you want `exclude_biomass` to exclude biomass reactions with `"EX_"` only? and not give the normal biomass function?

Reply:
Yeah, that's exactly my original intent. Basically, I wanted to find exchange reactions but on the off chance that there is a biomass exchange in the model, to be able to exclude that. If it's confusing, we can leave biomass out of it completely and I'll just do the filtering afterwards.

Reply:
Now we also have a `looks_like_biomass_reaction` function :)